doc/topics/releases/fluorine.rst

```diff
@@ -1315,3 +1315,13 @@ Proxy Modules
 - :mod:`salt.proxy.netmiko <salt.proxy.netmiko_px>`
 - :mod:`salt.proxy.nxos_api <salt.proxy.nxos_api>`
 - :mod:`salt.proxy.pyeapi <salt.proxy.arista_pyeapi>`
+
+Salt Cloud Features
```

Comment:
Hey @jgleissner - Apologies for not being clear about this before, but can you move these to the `neon.rst` release notes? That's the codename for the current `develop` branch.

Reply:
I moved it to `neon.rst`. Seems I misunderstood the version_number readme. Sorry about that. Thanks for your patience. :)